UnitTests/UnitTests.Shared/Mvvm/Test_ViewModelBase.cs

```diff
@@ -0,0 +1,107 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.Toolkit.Mvvm.Messaging;
+using Microsoft.Toolkit.Mvvm.Messaging.Messages;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.Mvvm
+{
+    [TestClass]
+    public class Test_ViewModelBase
```

Comment:
Should have tests here on Deactivation and the message registrations being removed.

Reply:
Right, added this in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/34af434d47e1ddb0051591477bbb009357db1407 👍